django_extensions/management/modelviz.py

```diff
@@ -210,7 +211,7 @@ def add_relation(field, extras=""):
                     related_query_name = field.related_query_name()
                     if verbose_names and related_query_name.islower():
                         related_query_name = related_query_name.replace('_', ' ').capitalize()
-                    label = '{} ({})'.format(label, force_bytes(related_query_name))
+                    label = '{} ({})'.format(label, force_text(related_query_name))
```

Comment:
To look more like Python3 probably `from __future__ import unicode_literals` should be put.
I have one concern about this for Python2 and for that reason I think that better approach is `u'{} ({})'` to work best on Python2 and Python3